tests/test_snapshots.py

```diff
@@ -39,7 +39,6 @@ def test_dict(snapshot, actual):
 
 def test_set(snapshot):
     assert snapshot == {"this", "is", "a", "set"}
-    assert snapshot == {"this", "is", "a", frozenset({"nested, set"})}
```

Comment:
A frozenset does not have a guaranteed order when it's serialized in pyyaml. A "set" itself can also not contain a set. I don't think this test adds much value in that case.

Reply:
Discussed offline, this is a limitation with yaml as a snapshot library should be able to handle any arbitrary data type